src/main/scala/edu/ie3/simona/model/thermal/CylindricalThermalStorage.scala

```diff
@@ -166,7 +166,7 @@ final case class CylindricalThermalStorage(
         return Some(lack)
       }
     }
-    None
```

Comment:
If you return in both cases Some(...) the Option in the return type is no longer necessary. 

But, as with our discussion, I would prefer not to touch this methods since it's not necessary.